src/BracketInserter.jl

```diff
@@ -33,6 +33,14 @@ function peek(b::IOBuffer)
     return c
 end
 
+# when we should not close the bracket
+function no_closing_bracket(left_peek, v)
+    # left_peek == v: we already have an open quote immediately before (triple quote)
+    # regex is for transposing calls: issue #200
+    # regex: immediately after letter/_: \w or closing array bracket \]
+    left_peek == v || (v == '\'' && occursin(r"\w|\]", string(left_peek)))
```

Comment:
Is it possible to not use a regex here? It feels quite "heavy handed".

Reply:
if the condition may be restricted to `'` only auto-completes when the proceeding character is a blank space then no regex is needed. In that case, typing <kbd>f='</kbd> will not auto-complete but [anything]<kbd>Space</kbd><kbd>'</kbd> will auto-complete. This is sort-of-ok since good programming practice is to have spaces in expressions like `foo = 'c'`. If it's fine, then this line may be changed to
```
left_peek == v || (v == '\'' && left_peek != ' ')
```

Reply:
> In that case, typing f=' will not auto-complete

It seems like you would want that to auto-complete.

Why is a regex needed otherwise? Can't it just be written out?

Reply:
Thanks for the feedback! I pushed a new commit to change to
```
isletter(left_peek) || isnumeric(left_peek) || left_peek == '_' || left_peek == ']'
```
to cover `foo'`, `foo3'`, `foo_'` and `[1 2 3]'`